gulpfile.js

```diff
@@ -151,158 +97,111 @@ gulp.task('clean-dev', function() {
         './dist/*.png',
         './dist/*.ico',
         './reports/**/*',
-        './reports']);
+        './reports'
+    ]);
 });
 
-gulp.task('clean-full', function() {
+gulp.task('clean-full', function () {
     del(['./dist/*',
         './reports/**/*',
-        './reports']);
+        './reports'
+    ]);
 });
 
 
 // =======================================================================
 // JSHint
 // =======================================================================
-gulp.task('lint', function() {
+gulp.task('lint', function () {
     return gulp.src(filePath.lint.src)
-    .pipe(jshint())
-    .pipe(jshint.reporter(stylish));
+        .pipe(jshint())
+        .pipe(jshint.reporter(stylish));
 });
 
 
 // =======================================================================
 // Javascript Checkstyles (JSCS)
 // =======================================================================
-gulp.task('checkstyle', function() {
+gulp.task('checkstyle', function () {
     return gulp.src(filePath.lint.src)
-    .pipe(jscs())
-    .on('error', handleError);
+        .pipe(jscs())
+        .on('error', handleError);
 });
 
 
 // =======================================================================
 // Browserify Bundle
 // =======================================================================
 
-// var bundle = {};
-// bundle.conf = {
-//     entries: filePath.browserify.src,
-//     external: filePath.vendorJS.src,
-//     debug: true,
-//     cache: {},
-//     packageCache: {}
-// };
-
-// function configureBundle(prod) {
-//     var bundler = browserify(bundle.conf);
-//
-//     if (!prod) {
-//         bundler = watchify(bundler);
-//         bundler.on('update', rebundle);
-//     }
-//
-//     bundler.transform(envify);
-//
-//     bundle.bundler = bundler;
-//     bundle.prod = prod;
-// }
-
-gulp.task('bundle-dev', function () {
-    // 'use strict';
-    // configureBundle(false);
-    // return rebundle();
-});
-
-gulp.task('bundle-prod', function () {
-    // 'use strict';
-    // configureBundle(true);
-    // return rebundle();
+gulp.task('bundle', function () {
+    return browserify({
+            entries: filePath.browserify.src,
+            debug: true
+        })
+        .transform(browserifyCss, {
+            global: true,
+        })
+        .transform(markedify)
+        .bundle() // Start bundle
+        .pipe(source(filePath.browserify.src)) // Entry point
+        .pipe(buffer()) // Convert to gulp pipeline
+        .pipe(sourceMaps.init({
+            loadMaps: true
+        })) // Strip inline source maps
+            .pipe(ngAnnotate())
+            .pipe(uglify({
+                mangle: false
```

Comment:
The total bundle is down to 2.33mb. Here's the biggest contributions by size after minification:
ag-grid: 585kb (a good candidate for elimination)
angular: 279kb
d3: 237kb (are we using this anywhere?)
angular-ui-bootstrap: 171kb
jquery: 129kb (?)
lodash: 124kb
swagger-client: 70kb (should be commented out at least)
appcode: 121kb

Maybe my next chunk of work will be removing some of these.